app/src/layouts/Layout.tsx

```diff
@@ -11,13 +11,20 @@ type LayoutProps = {
   children: ReactNode;
   rest?: unknown;
 };
+
+const pageTitle = atom("LinkUp");
+export const updateTitle = atom(null, (get, set, newTitle:string) =>
+  set(pageTitle, newTitle)
+);
```

Comment:
you might not need this `updateTitle`, instead you can just use the `pageTitle` atom